### PR TITLE
Add redfish as an enabled BIOS interface

### PR DESCRIPTION
**Pull request by bfournie**
Add Redfish as a BIOS interface to allow retrieving BIOS configuration
using Redfish.

### CODE REVIEW

File: ironic.conf [hunk 1]
```diff
@@ -5,6 +5,7 @@ default_boot_interface = ipxe
 default_deploy_interface = direct
 default_inspect_interface = inspector
 default_network_interface = noop
+enabled_bios_interfaces = no-bios,redfish
```
Comment (dtantsur) [inline ironic.conf:8]: Maybe enable the rest of them? At least `idrac-redfish` and `irmc` seem worth enabling

**Review by dtantsur — APPROVED** [review]
Looks good but needs rebasing.

**Review by iurygregory — APPROVED** [review]

**Pull request merged**